COPYRIGHT.txt

```diff
@@ -0,0 +1,14 @@
+
+Copyright 2021 The Sigstore Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Could you clarify why creating this fiile?
AFAIK in Apache License the convention is `NOTICE` file rather than `COPYRIGHT.txt`, see https://github.com/sigstore/sigstore-java/blob/27a57c87ea6c5cf6be4f64d824cd07f5656c67ac/LICENSE#L106

Reply:
oh yeah it was in the maven repo, must've left it in here while moving things around. We don't need this file. I'll remove it.